server/middlewares/isEmpty.js

```diff
@@ -1,13 +1,11 @@
 import { joinStrings } from '../helpers';
 
-const populateError = req => {
+const populateError = (req, ...fields) => {
   const error = [];
-  if (!req.body.name) {
-    error.push('name');
-  }
-  if (!req.body.hqAddress) {
-    error.push('hqAddress');
-  }
+  // eslint-disable-next-line array-callback-return
```

Comment:
Please use rules in .eslintrc file instead of specifying your rules in individual files.

Reply:
done!